hawc/apps/animal/models.py

```diff
@@ -1117,11 +1117,19 @@ def get_crumbs(self):
 
     def save(self, *args, **kwargs):
         # ensure our controlled vocabulary terms don't have leading/trailing whitespace
-        self.system = self.system.strip()
-        self.organ = self.organ.strip()
-        self.effect = self.effect.strip()
-        self.effect_subtype = self.effect_subtype.strip()
-        self.name = self.name.strip()
+        self.system = (
```

Comment:
@shapiromatron I noticed `save` was already overwritten on `Endpoint`, so I just went with that instead of making a pre-save signal. The formatting makes this look a little messy, lmk if you want me to restructure it so that it doesn't use ternary, just so it reads better.

Reply:
I changed it back to the way it was; so now you have the leeway of renaming non-terms manually through the form, but the API remains a little more strict in its usage.